pkg/cloud/gcp/actuators/services/compute/computeservice.go

```diff
@@ -101,3 +108,32 @@ func (c *computeService) TargetPoolsRemoveInstance(project string, region string
 func (c *computeService) MachineTypesGet(project string, zone string, machineType string) (*compute.MachineType, error) {
 	return c.service.MachineTypes.Get(project, zone, machineType).Do()
 }
+
+func (c *computeService) MachineTypesList(project string, zone string, ctx context.Context) (map[string]int64, []string) {
```

Comment:
Perhaps we should name this to identify that it is for MachineTypes that support guest accelerators?